Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -3789,6 +3789,15 @@ - (void)updateLastVisibleTimestamp:(uint64_t)timestamp
 
 - (void)markVisibleMessagesAsRead
 {
+    if (self.presentedViewController) {
+        OWSFail(@"%@ Not marking messages as read; another view is presented.", self.logTag);
+        return;
+    }
+    if (self.navigationController.topViewController != self) {
```

Comment:
This doesn't fix a bug - the behavior was already correct.  But this is important to get right so I want to future-proof us.